### PR TITLE
Changed more words to look more formal

### PR DESCRIPTION
**Pull request by vitoriasviana**

### CODE REVIEW

File: fator-previdenciario.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # -*- coding: UTF-8 -*-
 
 sexo = input("Informe o sexo [M/F]: ")
-idade = int(input("Digite a idade que você começou a trabalhar: "))
+idade = int(input("Digite a idade com a qual você começou a trabalhar: "))
```
Comment (eltonvs) [inline fator-previdenciario.py]: Please revert this change

Comment (vitoriasviana) [reply fator-previdenciario.py]: sorry

**Review by eltonvs — APPROVED** [review]
